lib/src/view/helper/chucker_ui_helper.dart

```diff
@@ -128,4 +129,8 @@ class ChuckerFlutter {
 
   ///[showChuckerScreen] navigates to the chucker home screen
   static void showChuckerScreen() => ChuckerUiHelper.showChuckerScreen();
+
+  ///[isNotificationEnabled] decides whether to show a snackbar message or
+  ///overlay notification when an API call is inspected.
+  static bool isNotificationEnabled = true;
```

Comment:
Duplicate of `ChuckerUiHelper.settings.showNotification`

Reply:
So I just made this property as because we can abstract the UiHelper class, avoiding the user to access other static properties

Reply:
not sure if you know it but there is already an option to turn off notification.
![image](https://github.com/syedmurtaza108/chucker-flutter/assets/44249868/ed7ff77f-66ea-485b-a28f-6aa0e5cd3637)

Reply:
@syedmurtaza108  Yeah I'm aware of it, but there should be a programmatical way to disable it. That's why I implemented it

Reply:
I agree with @cscipher , it is  really helpful if we can programmatically disable to show notifications without need to go chucker screen. Additionally, it would be great if we can set another parameter available in ChuckFlutter just like in https://pub.dev/packages/alice#additional-settings

Reply:
@cscipher This feature is implemented in release 1.6.0. You can now use `ChuckerFlutter.showNotification = false;` to disable notifications programmatically.